library/std/src/os/wasi/io/mod.rs

```diff
@@ -1,12 +1,6 @@
 //! WASI-specific extensions to general I/O primitives.
 
-#![deny(unsafe_op_in_unsafe_fn)]
 #![unstable(feature = "wasi_ext", issue = "71213")]
 
-mod fd;
-mod raw;
```

Comment:
This removed two `mod` statements but the corresponding files still exist. I think they are dead now? Or is there some magic going on that somehow loads them anyway?

Reply:
They're indeed dead now. I've now filed #135512 to  remove them.

Reply:
And I've now noticed that https://github.com/rust-lang/rust/pull/135491 already does this, so closing in favor off that.